index.html

```diff
@@ -510,6 +519,55 @@ <h3>Security and Privacy Considerations</h3>
+
+				<p>Resources within the Reading Order and Resource list that are non-audio resources should have a method for display, if possible for the User Agent. If the content cannot be displayed in the User Agent for any reason, it is recommended that the User Agent inform the user that the content is present by displaying it in the table of contents (if available), and provide a method for accessing it (see the section on <a href="#audio-ua-packaging">Packaging</a>.</p>
+
+				<p>If an audiobook has a <a href="#audio-pep">Primary Entry Page</a>, the User Agent can choose to display that as the Table of Contents.</p>
+
+			</section>
+
+			<section id="audio-ua-playback">
+				<h3>Audiobook Playability</h3>
+
+				<p>As outlined in the <a href="https://www.w3.org/TR/pwp-ucr">Use Cases and Requirements</a> note, an audiobook should be navigable in the User Agent. This means that a User Agent should provide methods for the user to move through the audiobook in a linear or non-linear fashion by either moving through the <a href="#audio-readingorder">Reading Order</a> seamlessly or by accessing the <a href="#audio-toc">Table of Contents</a>. The User Agent should also allow the user to move through individual audio files in short time increments.</p>
+
+				<p>For an audiobook, the User Agent should provide a <a href="https://www.w3.org/TR/pwp-ucr/#player">player interface</a> that will allow the user to navigate, play, or pause the audiobook. This interface can be represented to the user in any way (i.e. physical buttons, visual interface, keyboard input, or voice commands), but should be accessible at any point in the listening experience.</p>
+
+			</section>
+
+			<section id="audio-ua-packaging">
+				<h3>Audiobook Packaging and Offlining</h3>
+
+				<p>The <a href="https://www.w3.org/TR/pwp-ucr">Use Cases and Requirements</a> note recommends that content be available offline and that any packaged formats should not affect the iterations of the publications. This means that even if the content is copied many times to many users via multiple User Agents, the core manifest and it's identifier are never changed.</p>
+
+				<p>This specification recommends the <a href="https://www.w3.org/TR/lpf">Lightweight Packaging Format</a> for packaging audiobook content, but this is a requirement. Audiobook User Agents should be able to ingest LPF files for play, and should display content according to the requirements and recommendations in this document.</p>
+
```

Comment:
> but this is a requirement

I believe this should say

> but this is not a requirement

or is it really a requirement? In which case the usage of the term "recommends" may not be the right one...